srcpkgs/xonotic/template

```diff
@@ -25,6 +24,8 @@ do_build() {
 	esac
 	echo $make_build_args
 	cd source/darkplaces
+	sed -i makefile \
+		-e "s;DP_MACHINE:=.*;DP_MACHINE:=$XBPS_TARGET_MACHINE;"
```

Comment:
This expands to e.g. `x86_64-musl` and later checks won't pass as intended, you want `${XBPS_TARGET_MACHINE/-musl/}`